packages/ui/layouts/WizardLayout.tsx

```diff
@@ -43,6 +44,13 @@ export default function WizardLayout({
             <StepCard>{children}</StepCard>
           </div>
         </div>
+        {isOptionalCallback && (
+          <div className="mt-4 flex justify-center">
+            <Button color="minimal" onClick={isOptionalCallback}>
+              I&apos;ll do this later
```

Comment:
i18n string required

Reply:
Resolved in the [correct PR](https://github.com/calcom/cal.com/pull/9184) for these changes, thanks!